libraries/mbed/targets/hal/TARGET_RENESAS/TARGET_RZ_A1H/device.h

```diff
@@ -16,6 +16,10 @@
 #ifndef MBED_DEVICE_H
 #define MBED_DEVICE_H
 
+#if defined(__arm__) && (__ARMCC_VERSION <= 5040027)
```

Comment:
When we build our code in the web compiler, an inlining code was not generated correctly.
We contacted your Japanese personnel regarding this problem.
Then he guided this code to us and he replied to us as below.

"armcc version of Online compiler is ARM Compiler Version 5.03 Build 117.
 In case of "armcc 5.04 build 27 previous versions", inlining code may not be generated correctly when "-O3 -Otime" was specified."

He has confirmed to your compiler support team.